{{ cookiecutter.package_name }}/.coveragerc

```diff
@@ -0,0 +1,34 @@
+[run]
+omit = [
+ "{{ cookiecutter.module_name }}/_{{ cookiecutter._parent_project }}_init*",
+ "{{ cookiecutter.module_name }}/conftest.py",
+ "{{ cookiecutter.module_name }}/*setup_package*",
+ "{{ cookiecutter.module_name }}/tests/*",
+ "{{ cookiecutter.module_name }}/*/tests/*",
+ "{{ cookiecutter.module_name }}/extern/*",
+ "{{ cookiecutter.module_name }}/version*",
+ "*/{{ cookiecutter.module_name }}/_{{ cookiecutter._parent_project }}_init*",
+ "*/{{ cookiecutter.module_name }}/conftest.py",
+ "*/{{ cookiecutter.module_name }}/*setup_package*",
+ "*/{{ cookiecutter.module_name }}/tests/*",
+ "*/{{ cookiecutter.module_name }}/*/tests/*",
+ "*/{{ cookiecutter.module_name }}/extern/*",
+ "*/{{ cookiecutter.module_name }}/version*",
+]
+
+[report]
+exclude_lines = [
+  # Have to re-enable the standard pragma
+  "pragma: no cover",
+  # Don't complain about packages we have installed
+  "except ImportError",
+  # Don't complain if tests don't hit assertions
+  "raise AssertionError",
+  "raise NotImplementedError",
```

Comment:
Do these exclusions conflict with the logic of measuring coverage in tests @nabobalis ?

Reply:
Conflict in the sense they can increase the coverage if you omit them.

I think the import error one is useful since in our test suite we can never hit them. But I am a believer that any other raises should be hit by a test. You could argue for the `NotImplementedError` but I still think those should have a unit test.  

In theory if you had a large collection of ABC, I think without this, you would see a large drop in coverage?

Reply:
Why don't we keep them and then see what happens on core? We can always finesse them later?

Reply:
Sure.